src/CoreGraphics/CGContextPDF.cs

```diff
@@ -113,6 +117,12 @@ internal override NSMutableDictionary ToDictionary ()
 				ret.LowlevelSetObject (CFBoolean.FalseHandle, kCGPDFContextAllowsCopying);
 			if (AccessPermissions.HasValue)
 				ret.LowlevelSetObject (NSNumber.FromInt32 ((int) AccessPermissions.Value), kCGPDFContextAccessPermissions);
+			// only set the keys if they exists in the current OS version
+			if ((kCGPDFContextCreateLinearizedPDF != IntPtr.Zero) && CreateLinearizedPdf.HasValue)
+				ret.LowlevelSetObject (CFBoolean.ToHandle (CreateLinearizedPdf.Value), kCGPDFContextCreateLinearizedPDF);
+			// default to kCFBooleanFalse
+			if ((kCGPDFContextCreatePDFA != IntPtr.Zero) && CreatePdfA2u.HasValue && CreatePdfA2u.Value == true)
```

Comment:
Really minor, this can be simplified as:

```suggestion
			if ((kCGPDFContextCreatePDFA != IntPtr.Zero) && CreatePdfA2u == true)
```